react/components/WordpressCategory.tsx

```diff
@@ -77,8 +77,9 @@ const WordpressCategory: StorefrontFunctionComponent<CategoryProps> = ({
   const [loadingPage, setLoadingPage] = useState(true)
 
   useEffect(() => {
-    setLoadingPage(true)
-  }, [page])
+    query?.page && setPage(parseInt(query.page, 10))
+  }, [])

```

Comment:
```suggestion
  }, [query])

```
Just in case `query` is not populated for some reason when the component first loads, this will run the logic whenever `query` changes